conditional/blueprints/housing.py

```diff
@@ -118,6 +119,8 @@ def change_room_numbers(rmnumber):
             account = ldap_get_member(occupant)
             account.roomNumber = rmnumber
             log.info('api', action='%s assigned to room %s' % (occupant, rmnumber))
+            ldap_add_member_to_group(account, "active")
```

Comment:
Alternatively to the above comment, let's provide an API for toggling active status.